app/views/admin/products_v3/index.html.haml

```diff
@@ -32,7 +32,8 @@
     %tbody
       - @products.each do |product|
         %tr
-          %th.align-left=  product.name
+          %td.align-left
+            %b=  product.name
```

Comment:
Wouldn't it be better to use a css class  `.header` (or other name)  with `font-weight: bold;` ? With appropriately named class it gives us the indication this row is the header. 
The `<b>`  tag feels too naughty :smile:

Reply:
Yes, that's a better idea. 
I had hoped to use `th` which I think would be even better (potentially more semantically useful), but I don't know that there is any real reason to do so. 

👍 So your suggestion seems like the best approach.

Reply:
Fixed (although we will keep this on a development branch for the time being).